webpack.config.js

```diff
@@ -8,6 +8,7 @@ const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
```

Comment:
I think that the "webpack.config.js" file should be untracked from source control due to it being an autogenerated file that should not require any customisations.